src/rosdep2/platforms/arch.py

```diff
@@ -28,8 +28,9 @@
 # Author Tully Foote/tfoote@willowgarage.com
 
 import subprocess
```

Comment:
> Not entirely sure why the tests are failing now given that I copied existing code from `debian.py` to do this.

This should resolve the issue:

```suggestion
from __future__ import print_function
import subprocess
```